scripts/generate_psa_key_attributes/generate_psa_key_attributes.rst

```diff
@@ -0,0 +1,216 @@
+.. _generate_psa_key_attributes_script:
+
+PSA Key Attributes generator script
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This Python script generates Platform Security Architecture (PSA) key attribute binary blobs and creates JSON files for key provisioning with `nRF Util`_.
+
+.. note::
+   The script is primarily used for Key Management Unit (KMU) provisioning on nRF54L Series devices and nRF54H20.
+   For more details about KMU provisioning, see the :ref:`ug_nrf54l_developing_provision_kmu` in the nRF54L Series device guide and the :ref:`ug_nrf54h20_keys` in the nRF54H20 device guide.
+
+Overview
+********
+
+The script generates PSA key attributes that define the properties and usage of cryptographic keys.
+It supports multiple key sources and output formats:
+
+.. list-table:: Script Features
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Feature
+     - Supported options
+   * - Key sources
+     - | Raw HEX strings
+       | TRNG-generated keys
+       | PEM files
+   * - Output formats
+     - | :ref:`JSON files for nRF Util <generate_psa_key_attributes_script_nrfutil>`
+       | Binary metadata
+       | Standard output
+   * - Key types
+     - | AES
+       | HMAC
+       | ECC (various curves)
+       | ChaCha20
+       | Raw data
+   * - Storage locations
+     - | CRACEN
+       | CRACEN KMU
+
+The generated attributes include:
+
+* Key type and size
+* Storage location and persistence mode
+* Usage permissions (encrypt, decrypt, sign, verify, and more)
+* Cryptographic algorithms
+* CRACEN usage scheme (for KMU keys)
+
+.. _generate_psa_key_attributes_script_details:
+
+Implementation details
+======================
+
+The script creates PSA key attributes compatible with the ``psa_key_attributes_s`` C struct.
```

Comment:
Maybe add a link to the definition
https://github.com/nrfconnect/sdk-oberon-psa-crypto/blob/main/include/psa/crypto_struct.h#L286